finetune/model.py

```diff
@@ -141,16 +141,26 @@ def _model_fn(features, labels, mode, params):
 
         with tf.variable_scope(tf.get_variable_scope()):
             train_loss = 0.0
-            featurizer_state = params.base_model.get_featurizer(
-                X,
-                encoder=encoder,
-                config=params,
-                train=train,
-                explain=build_explain,
-            )
-            if context is not None:
-                print("Embedding position")
-                embed_position(context, featurizer_state, params, train)
+            if params.base_model.positional_info:
```

Comment:
I think all the featurizers should have **kwargs, so they should be able to take extra keyword arguments they don't use.

Reply:
Or, as the extra layers just work on the featurizer state anyway we can just put them in here and therefore be able to apply them to any basemodel?

Reply:
ah I like that!